javalin-graphql/src/main/java/io/javalin/plugin/graphql/GraphQLPlugin.kt

```diff
@@ -0,0 +1,29 @@
+package io.javalin.plugin.graphql
+
+import io.javalin.Javalin
+import io.javalin.core.plugin.Plugin
+import io.javalin.core.plugin.PluginLifecycleInit
+
+class GraphQLPlugin(private val options: GraphQLOptions) : Plugin, PluginLifecycleInit {
+
+    lateinit var graphQLHandler: GraphQLHandler
+
+    override fun init(app: Javalin) {
+        graphQLHandler = GraphQLHandler(options)
+    }
+
+    override fun apply(app: Javalin) {
+        app.get(options.path) {
+            it.contentType("text/html; charset=UTF-8")
+                    .result(
+                            GraphQLPlugin::class.java.getResourceAsStream("graphqli/index.html")
+                    )
+        }
+        app.post(options.path) { ctx ->
+            graphQLHandler.execute(ctx)
+        }
+        app.ws(options.path) { ws ->
```

Comment:
Can we add sse subscriptions? There are some usecases where sse are used over websockets. It is not the graphql default, but used sometimes.

Reply:
I like it. But, the query is sending with the path o by post method?

Reply:
I think we should start with just WebSocket, we can add SSE in a separate PR if there's a need later.